packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1398,15 +1398,21 @@ export class ContainerRuntime
 		});
 
 		const loadedFromSequenceNumber = this.deltaManager.initialSequenceNumber;
+		// If the base snapshot was generated when isolated channels were disabled, set the summary reference
+		// sequence to undefined so that this snapshot will not be used for incremental summaries. This is for
+		// back-compat and will rarely happen so its okay to re-summarize everything in the first summary.
+		const summaryReferenceSequenceNumber =
+			baseSnapshot === undefined || metadata?.disableIsolatedChannels === true
```

Comment:
I had no idea about these isolatedChannels. This seems like a really good catch

Reply:
I do not know what the solution is here, but we need somehow to tie our telemetry to decisions like that. I.e. if we see that we do a lot of full summaries, how do we learn what is the cause of it?